bin/Cores/cores.json

```diff
@@ -606,6 +611,12 @@
     "name": "Stella",
     "systems": [25]
   },
+  "swanstation_libretro":{
+    "name": "SwanStation",
+    "extensions": "m3u|cue",
+    "systems": [12],
+    "platforms": "none"
+  },
```

Comment:
Has there ever actually been a "swanstation_libretro" core? I thought SwanStation was just an internal naming thing and the file remained "duckstation_libretro".

Reply:
There was, yes. It changed for a while when both were present at the same time. The final version before it was pulled from the buildbot was under that name. There's a non-buildbot current version by that name as well.